### PR TITLE
React Capstone Project: Foot-Championships

### PR DESCRIPTION
**Pull request by sarazaToussaint**
### In this Pull Request i:

- I build the react app based on [ Azharimm ](https://github.com/azharimm/football-standings-api) API
-  Fetch a list of 20 leagues as categories and their teams as details from the API
- Create the tests for the application
- Style all components
- Deploy the project on Netlify
- Record a video for the project

### CODE REVIEW

File: src/components/__test__/Home.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      <Provider store={store}>
+        <BrowserRouter>
+          <Home />
+        </BrowserRouter>
+      </Provider>,
+    )
+      .toJSON();
+    expect(app).toMatchSnapshot();
+  });
+});
```
Comment (Gambit142) [inline src/components/__test__/Home.test.js:20]: - [ ] Great work using the renderer to test the components by checking the snapshot. To complete the tests for this project based on the requirements, kindly do the following:

- Write unit tests using Jest for some pure functions in your redux code
- Write integration tests for your React components using React Testing Library. Kindly read this [article](https://www.freecodecamp.org/news/8-simple-steps-to-start-testing-react-apps-using-react-testing-library-and-jest/) as a guide. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @sarazaToussaint 👋, 

Good job so far!

## Highlights
- Deployed project 💯 
- README.md file is professional 💯 
- Created the necessary components 💯 
- Uses react-router-dom 💯 
- Uses redux for state management 💯 
- Project follows design guidelines 💯 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- When using the `duck` pattern, it is common practice to not have the redux directory inside the components directory. 👍 

- You can also consider the other optional suggestions I made under the review. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by zhadier — APPROVED** [review]
## Hi @sarazaToussaint  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional
- [x] Great work implementing the search bar. But right now if I add something in the search bar and erase it, the filter does not go back and I can not see the entire list again. It would be great if you added that functionality as well so users don't have to refresh the page 👍 
<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/__test__/Home.test.js (hunk 1 above)
Comment (sarazaToussaint) [reply src/components/__test__/Home.test.js]: Hi @Gambit142, Thank you for reviewing my PR

**Comment by sarazaToussaint** [comment]
Hi @zhadier, thank you for approving my PR despite this issue with my filter search bar, I would like to inform you that I took my time to refactor my codes and fix the issue. Now the filter function is working perfectly.

**Pull request merged**